libcomposefs/lcfs-writer.c

```diff
@@ -564,7 +571,45 @@ int lcfs_node_set_fsverity_from_fd(struct lcfs_node_s *node, int fd)
 	return lcfs_node_set_fsverity_from_content(node, &_fd, fsverity_read_cb);
 }
 
-static int read_content(int fd, size_t size, uint8_t *buf)
+void lcfs_set_delayed_digest_calculation(struct lcfs_node_s *node)
+{
+	node->delayedDigestCalculation = true;
+}
+void lcfs_set_delayed_payload(struct lcfs_node_s *node)
+{
+	node->delayedPayload = true;
+}
+void lcfs_set_delayed_inline(struct lcfs_node_s *node)
+{
+	node->delayedInline = true;
+}
+
+void lcfs_set_reset_digest(struct lcfs_node_s *node)
+{
+	node->delayedResetDigest = true;
+}
+
+bool lcfs_get_delayed_digest_calculation(struct lcfs_node_s *node)
+{
+	return node->delayedDigestCalculation;
+}
+
+bool lcfs_get_delayed_payload(struct lcfs_node_s *node)
+{
+	return node->delayedPayload;
+}
+
+bool lcfs_get_delayed_inline(struct lcfs_node_s *node)
+{
+	return node->delayedInline;
+}
+
+bool lcfs_get_reset_digest(struct lcfs_node_s *node)
```

Comment:
Why is this not called get_delayed_reset_digest?

Reply:
That was a miss. These changes are not required anymore